draft-ietf-core-dns-over-coap.md

```diff
@@ -275,6 +278,62 @@ The construction algorithm for DoC requests is as follows, going through the pro
 
 A more generalized construction algorithm for any CoAP request can be found in {{-transport-indication}}.
 
+### Examples
+
+A typical SVCB resource record response for a DoC server at the root path "/" of the server looks
+like the following (the "docpath" SvcParam is the last 3 bytes "0a 00 00" in the binary):
```

Comment:
```suggestion
like the following (the "docpath" SvcParam is the last 4 bytes "00 0a 00 00" in the binary):
```

I'm no expert, but IIRC it's two 16-bit values, and that'd be consistent with the `00 01 00 03` value `02 63 6f` of a single 2-byte ALPN "co".